packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -644,7 +630,10 @@ export default {
                 fn.name.name
               }' definition into its own useCallback() Hook.`;
           }
+          // TODO: What if the function needs to change on every render anyway?
+          // Should we suggest removing effect deps as an appropriate fix too?
           context.report({
+            // TODO: Why not report this at the dependency site?
```

Comment:
I thought maybe it's worth reporting at both. But I want to report it here because you usually should start fixing it here. (Such as by either cutting it to paste later — outside component or inside effect, or by wrapping in useCallback.) I'm worried that reporting at the dependency site will make people think they're supposed to remove it from the array and silence the rule.